keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -40,14 +59,15 @@ def call(self, inputs):
         if self.norm_std:
             x = x / self._expand_non_channel_dims(self.norm_std, x)
 
-        return x
+        return x, y, sample_weight
```

Comment:
call `pack_x_y_sample_weight` here, it will gracefully handle the no label case.

Reply:
This still isn't done! Is there an issue with this one?